pandas/_libs/src/parser/tokenizer.c

```diff
@@ -507,7 +517,7 @@ static int end_line(parser_t *self) {
             TRACE((
                 "end_line: ERROR!!! self->lines(%zu) >= self->lines_cap(%zu)\n",
                 self->lines, self->lines_cap))
-            int bufsize = 100;
+            size_t bufsize = 100;
             self->error_msg = (char *)malloc(bufsize);
             snprintf(self->error_msg, bufsize,
```

Comment:
well if you like to open an issue that is fine, the original PR was closed and didn't have an issue.